packages/strapi-plugin-users-permissions/models/User.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  metadatas: {
+    resetPasswordToken: {
```

Comment:
should probably also add provider here

Reply:
Provider is currently there so I'll leave it as is for now :)

Reply:
I mean you shouldn't be able to edit it (or see it on the edit view)

Reply:
Yeah I guess you are right this just isn't used anyway in the controller. I'll remove it too then